src/org/traccar/protocol/CarcellProtocolDecoder.java

```diff
@@ -140,7 +140,7 @@ protected Object decode(
         if (parser.hasNext(4)) {
             position.set("cloned", parser.next().equals("1"));
 
-            parser.next(); // panic button status
+            position.set(Position.KEY_ALARM, parser.next()); // panic button status
```

Comment:
Reverted code as I am not prepared at this time to determine how to test the string to appropriately raise the alarm.